### DISCUSSION TITLE
Integration of moral models

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#716] [Research]
Hey all, 

Most of the things that we discussed this Friday in the research sync were in regard to the DDS project and surrounding questions/potential outputs. Here are some of the main ideas with a little more detail:

- As far as outputs/framing the project goes, we should really be considering the 'integration' of many different moral development 'models' (or ways of approaching moral development) for different contexts
- It is worth emphasizing here that many contexts/models have to be considered, and that there is no one moral development model

- So, the questions worth considering are: what constitutes a 'moral development model'? what are the different moral development models that we have been able to see in the organizations that we've researched/interviewed so far? What are some common links/differences in these models? How might they be 'integrated'?

- It remains an open-ended question exactly how we might want to do this, but one way to potentially do this is to look at George Kelly's notion of 'personal constructs' (an introduction of which can be found [here](https://www.verywellmind.com/george-kelly-biography-2795498)) and apply it to moral development. That would mean something like: our 'personal constructs' of the world, or 'ways of seeing the world' are tied to moral development/moral agency 
- It is also important to consider how these different models are applicable to co living 
- Another output is to potentially make a wiki which is able to organize some of this information 

Please feel free to expand further on anything else that might have been missed! See you guys Friday :)

**Comment by asimong** [comment]
See #700 for more on moral agency / development

**Comment by asimong** [comment]
> It remains an open-ended question exactly how we might want to do this, but one way to potentially do this is to look at George Kelly's notion of 'personal constructs' (an introduction of which can be found [here](https://www.verywellmind.com/george-kelly-biography-2795498)) and apply it to moral development.

I would like to see some delicacy here. Personally I see the moral (and related personal) development as a series of apparently slow-moving periods interspersed with what almost looks like quantum leaps of perspective / worldview / consciousness. The point of Kelly is to support the idea that there doesn't need to be just one way in which people do this, but rather, depending on the complexity of the world in which people live, they undergo personal construct reconstructions every so often. This *doesn't* mean that personal constructs are tied to moral development, but rather that moral development implies a shift in personal constructs.
